base/array.jl

```diff
@@ -2633,6 +2633,33 @@ function filter!(f, a::AbstractVector)
     return a
 end
 
+"""
+    filter(f)
+
+Create a function that filters its arguments with function `f` using [`filter`](@ref), i.e.
+a function equivalent to `x -> filter(f, x)`.
+
+The returned function is of type `Base.Fix1{typeof(filter)}`, which can be
+used to implement specialized methods.
+
+# Examples
+```jldoctest
+julia> (1, 2, Inf, 4, NaN, 6) |> filter(isfinite)
+(1, 2, 4, 6)
+
+julia> map(filter(iseven), [1:3, 2:4, 3:5])
+3-element Vector{Vector{Int64}}:
+ [2]
+ [2, 4]
+ [4]
+```
+!!! compat "Julia 1.8"
+    This method requires at least Julia 1.8.
```

Comment:
Should be 1.9 right?

Reply:
```suggestion
!!! compat "Julia 1.9"
    This method requires at least Julia 1.9.
```

Reply:
Was hoping that GitHub provides a way to make a new PR from a comment like this. Alas...

Reply:
Created the old fashioned way: https://github.com/JuliaLang/julia/pull/47045https://github.com/JuliaLang/julia/pull/47045.